src/templates/json.js

```diff
@@ -16,6 +16,22 @@ export default class JSONWrapper extends Component {
 		location: PropTypes.object
 	};
 
+	constructor (props) {
+		super(props);
+		this.state = {
+			responseRenderModuleDescription: null,
+			responseRenderModuleMembers: null
+		};
+	}
+
+	async componentDidMount () {
+		const doc = JSON.parse(this.props.data.jsonDoc.internal.content);
+		this.setState({
+			responseRenderModuleDescription: await renderModuleDescription(doc),
+			responseRenderModuleMembers: await renderModuleMembers(doc[0].members)
+		});
+	}
+
 	render () {
 		const doc = JSON.parse(this.props.data.jsonDoc.internal.content);
```

Comment:
As you are also using it in componentDidMount, you could add 'doc' to state and destructure it alongside 'responseRenderModuleDescription' and 'responseRenderModuleMembers' at the beginning of the render method.

Reply:
Now we have
`this.state = {
			renderDoc: null,
			responseRenderModuleDescription: null,
			responseRenderModuleMembers: null
		};`

and on `componentDidMount` we set the state as: 
`
this.setState({
			renderDoc: doc[0],
			responseRenderModuleDescription: await renderModuleDescription(doc),
			responseRenderModuleMembers: await renderModuleMembers(doc[0].members)
		});
`